### ISSUE TITLE
Enum syntax

### ISSUE THREAD

**Issue by machtelik** [Quick-Turn-Studio/CLionSupportForQt#13]
The following enum results in error `QmlTokenType.CLOSE_BRACE or QmlTokenType.COMMA expected, got '='`

```
    enum Alignment {
        Center = 0,
        Left
    }
```

**Comment by grabusr** [comment]
Hi Mike

Version 0.9.3 with a fix for that is available to download. Please confirm if it solves the issue.

**Comment by machtelik** [comment]
It's fixed in 0.9.3, thanks.
